components/more-less/more-less.js

```diff
@@ -48,6 +49,7 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 			inactive: { type: Boolean, reflect: true },
 			__blurBackground: { type: String },
 			__contentHeight: { type: String },
+			__maxHeight: { type: String },
```

Comment:
As of Lit 2, internal reactive state properties can be declared using Lit's `state`:
```javascript
__maxHeight: { state: true },
```
Feel free to update the others to this as well (`__blurBackground`, `__contentHeight`, `__transitionAdded`). This component was originally written pre-Lit 2.

Reply:
Change has been made, thanks!